src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is not the proper location for the file. You can try creating a dotnet new template by building the `Uno.ProjectTemplates.Dotnet` using the `pack` command in the IDE, then build an application using the `dotnet new` command. You'll be able to validate that the created app is able to be installed as a PWA.